src/core/config/descriptor_providers/xlio_config_schema.json

```diff
@@ -0,0 +1,1454 @@
+                                    "default": 16000,
+                                    "minimum": 0,
+                                    "title": "RX WRE global array size",
+                                    "description": "Number of Work Request Elements allocated in all RQs. Maps to XLIO_RX_WRE environment variable. "
+                                },
+                                "rx_batch_size": {
+                                    "type": "integer",
+                                    "default": 1024,
+                                    "minimum": 1,
+                                    "maximum": 1024,
+                                    "title": "RX WRE batch size",
+                                    "description": "Number of Work Request Elements and RX buffers to batch before recycling. Maps to XLIO_RX_WRE_BATCHING environment variable. Batching decrease latency mean, but might increase latency STD.\nValue range is 1-1024."
+                                }
+                            }
+                        },
+                        "tcp_segments": {
+                            "type": "object",
+                            "description": "TCP segmentation settings.",
+                            "properties": {
+                                "socket_batch_size": {
+                                    "type": "integer",
+                                    "default": 64,
+                                    "minimum": 1,
+                                    "title": "Socket segment batch size",
+                                    "description": "Number of TCP segments fetched from segments pool by a socket at once. Maps to XLIO_TX_SEGS_BATCH_TCP environment variable. Min value is 1"
+                                },
+                                "ring_batch_size": {
+                                    "type": "integer",
+                                    "default": 1024,
+                                    "minimum": 1,
+                                    "title": "Ring segment batch size",
+                                    "description": "Number of TCP segments fetched from segments pool by a ring at once. Maps to XLIO_TX_SEGS_RING_BATCH_TCP environment variable. Min value is 1"
+                                },
+                                "pool_batch_size": {
+                                    "type": "integer",
+                                    "default": 1024,
+                                    "minimum": 1,
+                                    "title": "Pool segment batch size",
+                                    "description": "Number of TCP segments batched when fetched from the segments pool. Maps to XLIO_TX_SEGS_POOL_BATCH_TCP environment variable. Min value is 1"
+                                }
+                            }
+                        }
+                    }
+                },
+                "max_gro_streams": {
```

Comment:
Should go to performance or protocol.tcp

Reply:
Agreed, it's under performance